### PR TITLE
added initial pwa support via docusaurus plugin

### PR DESCRIPTION
**Pull request by jadonwolffs**
As per issue #34 I have added the PWA plugin to the project and followed the Docusaurus guide to configure it

If you load the page (using `npm run build` and `npm run serve` because it only activates the plugin during a production build) with the developer console open you can see that it does spawn a service worker and work to cache the assets but it doesn't pop up with an option to install the page

### CODE REVIEW

**Comment by lelouchB** [comment]
Hey @jadonwolffs 
I will look into this and get back to you shortly.

Thanks

**Comment by lelouchB** [comment]
Hey @jadonwolffs 
Few things : 
-  Move the plugins code from top to bottom in `docusaurus.config.js`
- Add few additonal details in `manifest.json`, 
```js
    "lang": "en",
    "start_url": "https://finalspaceapi.com",
```
-  I guess the problem is with images, they are mandatory in `manifest.json`. We need to generate square images for this.

image

I understand with the current logo, generating square pics of various dimensions will not be possible. There is seperate for logo and people are working on it.

For now, use this image to generate various dimensions square images and store them in a seperate directory in `static/img` folder
Use this tool https://www.pwabuilder.com/imageGenerator
image



Resources: 
 https://developer.mozilla.org/en-US/docs/Web/Manifest
https://developer.mozilla.org/en-US/docs/Web/Manifest/icons

**Comment by jadonwolffs** [comment]
Woah, nice - I made these changes and it seems to work nicely! Thanks for helping me through this @lelouchB

File: frontend/docusaurus.config.js [hunk 1]
```diff
@@ -83,10 +83,6 @@ module.exports = {
       ],
       copyright: "Handcrafted by me © twentytwenty",
     },
-    googleAnalytics: {
```
Comment (lelouchB) [inline frontend/docusaurus.config.js:86]: Please add this code block in your PR

Comment (jadonwolffs) [reply frontend/docusaurus.config.js:86]: should be done now

**Review by lelouchB — APPROVED** [review]

**Comment by lelouchB** [comment]
> 
> 
> Woah, nice - I made these changes and it seems to work nicely! Thanks for helping me through this @lelouchB

Hey @jadonwolffs 
Glad I could help, I was shipping this change to production.
It's live.
 Also, here is a small blog on this https://finalspaceapi.com/blog/pwa/

Thanks for your valuable contributions. 💯

**Pull request merged**